src/App.tsx

```diff
@@ -21,14 +90,14 @@ export const App: React.FC = () => {
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {/* <Loader /> */}
+              {needLoader && (<Loader />)}
+              <TodoList list={newList()} />
```

Comment:
```suggestion
              <TodoList todos={newList()} />
```
list doesn't tell what exactly it is (similar to `array` - bad name)

Reply:
it means done )